HelpMyStreetFE/HelpMyStreetFE/src/sass/views/_content-page.scss

```diff
@@ -36,6 +36,10 @@
         li:before {content: counters(item, ".") " "; counter-increment: item}
     }
 
+  &__body {
+      font-size: 16px;
```

Comment:
We have a rule `* { font-size: 18px; }` (which I dislike, and) which will apply to all child elements of `.content-page__body`.  I therefore wouldn't expect this new rule to have any effect.

If you want to override this you'd need `content-page__body * { font-size: 16px; }` - but bear in mind the Resources page has this class too, and you probably don't want to change the size of the headings there.  Maybe define a more specific class `.body-text-16px`, with css something like the following:

`
/* Applies to any p, ol or ul which is a child of a .body-text-16px, and additionally and children of those elements */
.body-text-16px {
  p, p *, ol, ol *, ul, ul * {
    font-size: 16px;
  }
}
`

(Apparently line breaks don't persist in this thing, sorry!)

Reply:
![image](https://user-images.githubusercontent.com/58040070/96293185-46378000-0fe2-11eb-97f0-2bbfb0060347.png)